### PR TITLE
pegasus lvs

### PR DESCRIPTION
**Pull request by jjthomas**

### CODE REVIEW

**Comment by steveri** [comment]
Does this work on local machines, or just vde (i.e. do we have pegasus installed in /cad)?

**Comment by alexcarsello** [comment]
@steveri This will only work on VDE. We don't have Pegasus on our machines.

File: steps/cadence-pegasus-lvs/run.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+if [ ! -z "$lvs_hcells_file" ]; then hcell="-hcell $lvs_hcells_file"; fi
+black_box=""
+if [ -e inputs/rules.svrf ]; then
+  while read line; do
+    if [[ $line == *"LVS BOX"* ]]; then
+      arr=($(echo $line | xargs))
+      black_box="${black_box}lvs_black_box ${arr[-1]} -source_layout;\n"
+    fi
+  done < inputs/rules.svrf
+fi
```
Comment (alexcarsello) [inline steps/cadence-pegasus-lvs/run.sh:10]: What's going on here? Are you trying to basically convert the Calibre LVS BOX commands to something Pegasus can support?

**Review by alexcarsello — APPROVED** [review]
I think this looks good, just the question on the black box stuff, but I think I understand what's going on. Have you tested this node on VDE already?

File: steps/cadence-pegasus-lvs/run.sh (hunk 1 above)
Comment (jjthomas) [reply steps/cadence-pegasus-lvs/run.sh]: Yes, that's right

**Comment by mbstrange2** [comment]
@jjthomas @alexcarsello can we merge this?

**Comment by alexcarsello** [comment]
As long as @jjthomas says it's good to go then yes

**Pull request merged**